### PR TITLE
MB-8141 increase jest coverage threshold

### PR DESCRIPTION
**Pull request by jenniferlynparsons**
## Description

I added a number of loading and error component tests, fixed a small bug, and bumped up our Jest minimum coverage numbers.

## Reviewer Notes

There are definitely ways we can streamline these tests further. I considered putting the loading/error component tests at the bottom for legibility and to keep focus on the rest of the tests. I also considered not creating `const` for the loading and error data since it's streamlined but opted not to for the moment. Style-wise, this means keeping all the return data at the top of the tests and I cleaned up a few other similar instances as well.

I would have loved to take time to clean these up further. Many of the loading/error return values had to include the success return data as well or there were `undefined` failures. This is a smell for both how our code is handling data and also how and what our tests are testing.

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make server_run
make office_client_run

yarn test
```

## Code Review Verification Steps

* [ ] If the change is risky, it has been tested in experimental before merging.
* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#logging)
* [ ] The requirements listed in [Querying the Database Safely](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#querying-the-database-safely) have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/wiki/migrate-the-database#zero-downtime-migrations))
  * [ ] Have been communicated to #g-database
  * [ ] Secure migrations have been tested following the instructions in our [docs](https://github.com/transcom/mymove/wiki/migrate-the-database#secure-migrations)
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-8141) for this change

### CODE REVIEW

File: src/pages/Office/CustomerInfo/CustomerInfo.test.jsx [hunk 1]
```diff
@@ -45,45 +45,94 @@ const mockCustomer = {
...
+    });
+  });
+
+  it('populates initial field values', async () => {
     render(
       <MockProviders initialEntries={[customerInfoEditURL]}>
-        <CustomerInfo customer={mockCustomer} ordersId="abc123" isLoading={false} isError={false} />{' '}
+        <CustomerInfo
+          customer={mockCustomer}
+          onUpdate={mockUpdate}
```
Comment (jenniferlynparsons) [inline src/pages/Office/CustomerInfo/CustomerInfo.test.jsx:96]: This was missing and causing error messages.

File: src/pages/Office/CustomerInfo/CustomerInfo.test.jsx [hunk 2]
```diff
@@ -45,45 +45,94 @@ const mockCustomer = {
   phone: '123-444-3434',
 };
 
+const mockUpdate = jest.fn();
```
Comment (jenniferlynparsons) [inline src/pages/Office/CustomerInfo/CustomerInfo.test.jsx:48]: Moved this here so it's not repeated throughout the tests.

File: src/pages/Office/MoveDetails/MoveDetails.test.jsx [hunk 3]
```diff
@@ -515,9 +516,54 @@ const approvedMoveDetailsQuery = {
...
+        </MockProviders>,
+      );
+
+      const errorMessage = await screen.getByText(/Something went wrong./);
+      expect(errorMessage).toBeInTheDocument();
+    });
+  });
   describe('requested shipment', () => {
-    useMoveDetailsQueries.mockImplementation(() => requestedMoveDetailsQuery);
+    useMoveDetailsQueries.mockReturnValue(requestedMoveDetailsQuery);
```
Comment (jenniferlynparsons) [inline src/pages/Office/MoveDetails/MoveDetails.test.jsx:566]: There's no need for an mock implementation if we're not doing anything within that mock. We just need the returned value.

File: src/pages/Office/CustomerInfo/CustomerInfo.test.jsx [hunk 4]
```diff
@@ -45,45 +45,94 @@ const mockCustomer = {
...
-    expect(screen.getByDisplayValue(mockCustomer.email).value).toEqual(mockCustomer.email);
-    expect(screen.getByDisplayValue(mockCustomer.backup_contact.email).value).toEqual(
-      mockCustomer.backup_contact.email,
-    );
-    expect(screen.getByLabelText('Address 1').value).toEqual(mockCustomer.current_address.street_address_1);
-    expect(screen.getByLabelText('City').value).toEqual(mockCustomer.current_address.city);
-    expect(screen.getByLabelText('State').value).toEqual(mockCustomer.current_address.state);
-    expect(screen.getByLabelText('ZIP').value).toEqual(mockCustomer.current_address.postal_code);
-    expect(screen.getByLabelText('Name').value).toEqual(mockCustomer.backup_contact.name);
+    await waitFor(() => {
```
Comment (jenniferlynparsons) [inline src/pages/Office/CustomerInfo/CustomerInfo.test.jsx:103]: This resolves those pesky Formik errors. A deeper dive into why this is necessary would be useful, but I'm trying not to go tooo far down the rabbit hole.

**Review by Ryan-Koch — APPROVED** [review]

**Pull request merged**